keyboards/romac/romac.h

```diff
@@ -0,0 +1,18 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
QMK formatting suggests the usage of LAYOUT over KEYMAP when referring to layout macros. 

```suggestion
#define LAYOUT( \
```